pkg/tfshim/sdk-v2/cty_test.go

```diff
@@ -821,6 +826,12 @@ func normalizeCtyValue(v cty.Value) cty.Value {
 			f.SetPrec(512)
 			return cty.NumberVal(f), nil
 		}
+		if v.Type().IsSetType() && v.IsNull() {
```

Comment:
What would it take to not have these changes? This significantly weakens the tests as it says that the tests don't care anymore about the difference between null and empty sets.